FWCore/ServiceRegistry/interface/ActivityRegistry.h

```diff
@@ -28,11 +28,16 @@ to this file that go beyond the obvious cut and paste type of edits.
   1. The number at the end of the AR_WATCH_USING_METHOD_X macro definition
   is the number of function arguments. It will not compile if you use the
   wrong number there.
-  2. Use connect or connect_front depending on whether the callback function
+  2. Inside the watch function definition, choose either connect or
+  connect_front depending on whether the callback function
   should be called for different services in the order the Services were
-  constructed or in reverse order. Begin signals are usually forward and
-  End signals in reverse, but if the service does not depend on other services
-  and vice versa this does not matter.
+  constructed or in reverse order. If services don't depend on one another,
+  this choice does not matter (most services fall in this category, maybe
+  all). There is some inconsistency in how this has been implemented. Originally,
```

Comment:
There are Services that depend on other Service(s), so I'd remove "maybe all".

Reply:
I heavily reworded the part of the comment in bullet 2.